src/Fixer/DescribedFixerInterface.php

```diff
@@ -10,13 +10,20 @@
  * with this source code in the file LICENSE.
  */
 
-namespace PhpCsFixer\ConfigurationException;
+namespace PhpCsFixer\Fixer;
+
+use PhpCsFixer\FixerDefinitionInterface;
 
 /**
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
- *
- * @internal
+ * @author SpacePossum
  */
-final class UnallowedFixerConfigurationException extends InvalidFixerConfigurationException
+interface DescribedFixerInterface
```

Comment:
hm... in our repo all fixers should be described.
not sure is it a good idea to allow 3rd party fixers to not be described.
they would work, but it's nice to enforce quality of user experience